recipe/meta.yaml

```
@@ -21,101 +21,18 @@ requirements:
     - pip
   run:
     - python
-    - langchain-core >=0.1.52,<0.2.0
-    - sqlalchemy >=1.4,<3
+    - langchain <0.4.0,>=0.3.11
+    - langchain-core <0.4.0,>=0.3.24
+    - sqlalchemy <3,>=1.4
     - requests >=2.0.0,<3.0.0
     - pyyaml >=5.3
     - numpy >=1.0.0,<2.0.0
```

https://github.com/langchain-ai/langchain/blob/langchain-community%3D%3D0.3.12/libs/community/pyproject.toml#L44-L50

```suggestion
    - numpy >=1.22.4,<2.0.0  # [py<312]
    - numpy >=1.26.2,<3.0.0  # [py>=312]
```